### PR TITLE
Сheck applyByNode nodeNum arg

### PR DESCRIPTION
**Pull request by msaf1980**
Сheck applyByNode nodeNum arg for prevent overflow

### CODE REVIEW

File: webapp/graphite/render/functions.py [hunk 1]
```diff
@@ -5102,7 +5102,10 @@ def applyByNode(requestContext, seriesList, nodeNum, templateFunction, newName=N
   """
   prefixes = set()
   for series in seriesList:
-    prefix = '.'.join(series.name.split('.')[:nodeNum + 1])
+    nodes = series.name.split('.')
+    if nodeNum >= nodes.length:
```
Comment (deniszh) [inline webapp/graphite/render/functions.py:5106]:
```suggestion
    if nodeNum >= len(nodes):
```

File: webapp/graphite/render/functions.py [hunk 2]
```diff
@@ -5102,7 +5102,10 @@ def applyByNode(requestContext, seriesList, nodeNum, templateFunction, newName=N
   """
   prefixes = set()
   for series in seriesList:
-    prefix = '.'.join(series.name.split('.')[:nodeNum + 1])
+    nodes = series.name.split('.')
+    if nodeNum >= len(nodes):
```
Comment (deniszh) [inline webapp/graphite/render/functions.py:5106]:
```suggestion
    if nodeNum > len(nodes):
```

Comment (msaf1980) [reply webapp/graphite/render/functions.py]: My mistake. List in Python have len() method. This fix (len function) for Golang.

**Comment by deniszh** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|1.1.x|[](https://github.com/graphite-project/graphite-web/pull/2803)|


### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**